panda/src/pnmimage/pnmimage_base.cxx

```diff
@@ -33,8 +33,8 @@ pm_message(const char *format, ...) {
 
   static const size_t buffer_size = 1024;
   char buffer[buffer_size];
-#if defined(WIN32_VC) || defined(WIN64_VC)
-  // Windows doesn't define vsnprintf().  Hope we don't overflow.
+#ifdef _MSC_VER
+  // Visual Studio doesn't define vsnprintf().  Hope we don't overflow.
```

Comment:
It seems like [this is no longer the case](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/vsnprintf-vsnprintf-vsnprintf-l-vsnwprintf-vsnwprintf-l?view=vs-2015); getting rid of the special-case altogether sounds worth trying.